src/Dropdown.php

```diff
@@ -18,54 +18,50 @@ class Dropdown extends Lister
     public $cb;
 
     /**
-     * Supply an optional parameter to the drop-down.
+     * As per Fomantic-ui dropdown options.
      *
-     * @var array will be converted to json passed into dropdown()
+     * @var array
      */
-    public $js;
+    public $dropdownOptions = [];
 
     protected function init(): void
     {
         parent::init();
 
         if (!$this->cb) {
-            $this->cb = JsCallback::addTo($this, ['urlTrigger' => 'item']);
+            $this->cb = JsCallback::addTo($this);
         }
     }
 
     /**
      * Handle callback when user select a new item value in dropdown.
      * Callback is fire only when selecting a different item value then the current item value.
      * ex:
-     *      $dropdown = Dropdown::addTo($menu, ['menu', 'js' => ['on' => 'hover']]);
+     *      $dropdown = Dropdown::addTo($menu, ['menu', 'dropdownOptions' => ['on' => 'hover']]);
      *      $dropdown->setModel($menuItems);
      *      $dropdown->onChange(function($item) {
-     *          return 'New seleced item: '.$item;
+     *          return 'New selected item: '.$item;
      *      });.
      *
      * @param \Closure $fx handler where new selected Item value is passed too
      */
-    public function onChange(\Closure $fx)
+    public function onChange(\Closure $fx, string $argumentName = 'item')
```

Comment:
when a different argument name makes sense?

Reply:
Let's say you have two dropdowns in the same page. They will both use 'item' in the post request. This will help differentiate them.

Reply:
you mean post request of dynamic dropdown?

Reply:
I was thinking more in term of handling multiple dropdown using one function, something like:
```
$menu_selected = function($id) {
  $menu = isset($_POST['menu_1']) ? 'menu 1': (isset($_POST['menu_2']) ? 'menu 2' : 'unknown');

  return 'you have selected item ' . $id . ' of ' . $menu;
};

$menu = \atk4\ui\Menu::addTo($app);
$dropdown = \atk4\ui\Dropdown::addTo($menu, ['Menu 1', 'dropdownOptions' => ['on' => 'hover']]);
$dropdown->setSource($items);
$dropdown->onChange($menu_selected, 'menu_1');

$dropdown = \atk4\ui\Dropdown::addTo($menu, ['Menu 2', 'dropdownOptions' => ['on' => 'hover']]);
$dropdown->setSource($items);
$dropdown->onChange($menu_selected, 'menu_2');
```

By default onChange handler is using POST request, may be we could change this for GET instead?

Reply:
we should definitely handle this differently, so drop it

what you want is to simply pass a different callback like:
```
$dropdown->onChange(function ($id) use ($menu_selected) { $menu_selected($id, 'menu_1'); });
```

Reply:
make sense!